apps/easypid/src/features/receive/DeferredCredentialNotificationScreen.tsx

```diff
@@ -0,0 +1,153 @@
+    icon: <HeroIcons.Trash />,
+    onPress: withHaptics(() => setIsSheetOpen(true)),
+    renderCondition: true,
+  })
+
+  const credentialDisplay = getCredentialDisplayWithDefaults(
+    getOpenId4VcCredentialDisplay(
+      extractOpenId4VcCredentialMetadata(response.credentialConfiguration, {
+        display: issuerMetadata.credentialIssuer?.display,
+        id: issuerMetadata.credentialIssuer?.credential_issuer,
+      })
+    )
+  )
+
+  return (
+    <>
+      <FlexPage p={0} gap={0}>
+        <YStack
+          w="100%"
+          top={0}
+          p="$4"
+          borderBottomWidth="$0.5"
+          borderColor={isScrolledByOffset ? '$grey-200' : '$background'}
+        />
+        <ScrollView onScroll={handleScroll} scrollEventThrottle={scrollEventThrottle}>
+          <YStack ai="center" gap="$6" p="$4" marginBottom={bottom}>
+            <AnimatedStack width="100%" mt="$-3" mb="$-5" scale={0.75}>
+              <FunkeCredentialCard
+                issuerImage={{
+                  url: credentialDisplay.issuer.logo?.url,
+                  altText: credentialDisplay.issuer.logo?.altText,
+                }}
+                textColor={credentialDisplay.textColor}
+                name={credentialDisplay.name}
+                backgroundImage={{
+                  url: credentialDisplay.backgroundImage?.url,
+                  altText: credentialDisplay.backgroundImage?.altText,
+                }}
+                isLoading={true}
+                bgColor={credentialDisplay.backgroundColor ?? '$grey-900'}
+              />
+            </AnimatedStack>
+            <Stack gap="$2">
+              <Heading ta="center" heading="h1">
+                <Trans id="deferredCredentials.title" comment="Title of the deferred credential notification screen">
+                  Deferred credential
+                </Trans>
+              </Heading>
+              <Paragraph numberOfLines={2} ta="center">
+                <Trans id="common.issuedBy" comment="Prefix before issuer name">
+                  Issued by {credentialDisplay.issuer.name}.
+                </Trans>
+              </Paragraph>
+            </Stack>
+            <YStack w="100%" gap="$2">
+              {deferredCredential.lastErroredAt && (
+                <InfoButton
+                  variant="danger"
+                  title={t({
+                    id: 'deferredCredentials.failedToRetrieveCard',
+                    message: 'Failed to retrieve card',
+                    comment: 'Title of the error box explaining deferred credentials',
+                  })}
+                  description={t({
+                    id: 'deferredCredentials.failedToRetrieveCardDescription',
+                    message: 'An error occurred while retrieving the deferred card from the issuer.',
+                    comment: 'Description of the error box explaining deferred credentials',
+                  })}
+                />
+              )}
+              <InfoButton
```

Comment:
I think it would be good to share the `interval` with the user. E.g. `The issuer of the card indicated you should check back in 2 hours`